mne/viz/topo.py

```diff
@@ -365,11 +357,11 @@ def _format_coord(x, y, labels, ax):
         timestr = '%6.3f s: ' % x
         if not nearby:
             return '%s Nothing here' % timestr
+        labels = [''] * len(nearby) if labels is None else labels
```

Comment:
moving this up from line 372, otherwise `labels[n]` on next line fails when `labels` is `None`